pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -44,7 +44,7 @@
 , statix
 , stylish-haskell
 , tabnine
-, taskwarrior
+, taskwarrior3
```

Comment:
Several, if not many, people seem to prefer to use Git to manage task data. For example, see [this discussion](https://www.reddit.com/r/taskwarrior/comments/1dffv6b/should_i_use_task_warrior_v3_or_v2/). The current option is sticking to TaskWarrior 2. Perhaps instead of replacing `taskwarrior` with `taskwarrior3`, keeping both of them is a better option?

Reply:
> Several, if not many, people seem to prefer to use Git to manage task data. For example, see [this discussion](https://www.reddit.com/r/taskwarrior/comments/1dffv6b/should_i_use_task_warrior_v3_or_v2/). The current option for them is to stick with TaskWarrior 2. Therefore, perhaps instead of replacing `taskwarrior` with `taskwarrior3`, keeping both of them is better here?

The vim plugin that is evaluated there eventually is based on the `ftplugin` and `ftdetect` and `syntax` files of Taskwarrior - simply meant to give you syntax highlighting for the `.taskrc` file. These files were barely changed between 2.6 -> 3.0, so I don't see a reason to put them both.

Reply:
> The change is not trivial either:
> 
> ```
> $ diff /nix/store/m99g4v6xj29yb8mgqkc7v90lcrbk26di-taskwarrior-3.0.2/share/vim-plugins/task/syntax/taskrc.vim /nix/store/p32chw38kv19i3lnx85fsxcalbjdd9yf-taskwarrior-2.6.2/share/vim-plugins/task/syntax/taskrc.vim | wc -l
> 58
> ```
> 
> The change may become bigger in the future, too.

Hmm that is convincing. I added the attributes `vimPlugins.taskwarrior{2,3}` and made `vimPlugins.taskwarrior` a throwing alias.